antsibull/cli/antsibull_build.py

```diff
@@ -16,22 +16,27 @@
 from ..app_logging import log
 from ..args import InvalidArgumentError, get_common_parser, normalize_common_options
 from ..config import load_config
-from ..new_acd import new_acd_command
+from ..new_ansible import new_ansible_command
 from ..build_collection import build_collection_command
-from ..build_acd_commands import build_single_command, build_multiple_command
+from ..build_ansible_commands import build_single_command, build_multiple_command
 from ..build_changelog import build_changelog
 
 
 mlog = log.fields(mod=__name__)
 
-DEFAULT_FILE_BASE = 'acd'
+DEFAULT_FILE_BASE = 'ansible'
 DEFAULT_PIECES_FILE = f'{DEFAULT_FILE_BASE}.in'
 
-ARGS_MAP = {'new-acd': new_acd_command,
+ARGS_MAP = {'new-ansible': new_ansible_command,
+            'single': build_single_command,
+            'multiple': build_multiple_command,
+            'collection': build_collection_command,
+            'changelog': build_changelog,
+            # Old names, deprecated
+            'new-acd': new_ansible_command,
```

Comment:
Are the old names still needed? I don't see that they can still be used.

Reply:
Hmmm..... I'm going to see if I can get that backwards compat working.. If not, then it isn't needed.